dandiapi/settings.py

```diff
@@ -65,6 +65,7 @@ class DevelopmentConfiguration(DandiMixin, DevelopmentBaseConfiguration):
 
 class TestingConfiguration(DandiMixin, TestingBaseConfiguration):
     MINIO_STORAGE_MEDIA_BUCKET_NAME = 'test-django-storage'
+    MINIO_STORAGE_MEDIA_URL = 'http://minio:9000/test-django-storage'
```

Comment:
```suggestion
    MINIO_STORAGE_MEDIA_URL = 'http://localhost:9000/test-django-storage'
```
This fixes the tests for me locally. @waxlamp do you recall why we needed the container name in the `TestingConfiguration`? I would think it would only be necessary when running in development.